test/deploy/linux/elasticsearch/install/linux2/roles/configure/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
@setamyDG please replace the linux2 in the path to rhel, since this ansible play will/should also work on redhat and centos.